css-values-4/Overview.bs

```diff
@@ -1965,9 +1965,9 @@ Combination of <<color>></h4>
 	(that is, colors for which the red, green, and blue components specified
 	have been multiplied by the alpha).
 
-	<a>Addition</a> of <<number>> is likewise defined as
+	<a>Addition</a> of <<color>> is likewise defined as
 	the independent <a>addition</a> of each component
-	as a <<number>>
```

Comment:
This one should stay as number, since it's talking about the value of a color component.